src/main/java/de/zalando/aruha/nakadi/domain/BatchItemResponse.java

```diff
@@ -3,7 +3,7 @@
 public class BatchItemResponse {
     private EventPublishingStatus publishingStatus = EventPublishingStatus.ABORTED;
     private EventPublishingStep step = EventPublishingStep.NONE;
-    private String detail;
+    private String detail = "";
```

Comment:
I believed in this answer http://stackoverflow.com/a/1171707 it seems that Java "interns" string literals so to reuse them. I didn't experimented but found plenty examples all over the internet.